### PR TITLE
Low zoom translations

### PR DESCRIPTION
**Pull request by zerebubuth**
Here's a first attempt at solving #977. I've included translations in the `places` and `water` layers, where available. For example:

output

It looks like we're not using any data from the physical labels theme. Is that new in v4, or are we getting that data from OSM? Let me know what you think, and let's file follow-up issues for bringing in any new (or missing) data we might want.

Connects to #977.

### CODE REVIEW

File: data/schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
 -- wof
 
+-- drop table before creating so that we can run this over and over. this
+-- can be useful when the script fails part of the way through and needs to
+-- be re-run.
+DROP TABLE IF EXISTS wof_neighbourhood_placetype CASCADE;
```
Comment (nvkelso) [inline data/schema.sql]: 👍

File: scripts/mktest.py [hunk 2]
```diff
@@ -271,6 +272,9 @@ def naturalearth_test(args):
     ne_test_parser.add_argument(
         '--ne-id', type=int, default=1,
         help='Natural Earth ID to use.')
+    ne_test_parser.add_argument(
+        '--layer-name', default='pois',
```
Comment (nvkelso) [inline scripts/mktest.py]: Nit: Not many NE features ending up in `pois` layer, maybe `places` is better?

File: vectordatasource/meta/sql.py [hunk 3]
```diff
@@ -671,7 +671,7 @@ def table_is_polygonal(name):
 
 
 def calculate_meta_source_for_table(name):
-    if table_is_osm(name):
+    if table_is_osm(name) or name == 'admin_areas':
```
Comment (nvkelso) [inline vectordatasource/meta/sql.py]: good catch

File: data/apply-schema-update.sql [hunk 4]
```diff
@@ -1,94 +1,113 @@
+CREATE OR REPLACE function tmp_add_column(tbl_name text, cln_name text, typ_name text)
```
Comment (nvkelso) [inline data/apply-schema-update.sql]: Can you explain the name of this function more? Seems like the column is persistent so seems like a weird name.

**Comment by nvkelso** [comment]
There are a few followup issues that are already in the milestone (and we can add more), for physical labels and others.

I think the next step is `country` and `region` labels, and the min_zoom, max_zoom in a new PR.

File: scripts/mktest.py (hunk 2 above)
Comment (zerebubuth) [reply scripts/mktest.py]: Good point. Fixed in f585eff.

**Pull request merged**